docs/API.md

```diff
@@ -3,7 +3,7 @@
 ## OneDrive
 Helper class that facilitates accessing one drive.
 
-**Kind**: global class  
```

Comment:
My editor automatically removes empty trailing space, please tell me if it shouldn't for that file.